shed/event_streams.py

```diff
@@ -459,7 +460,7 @@ def stop(self, docs):
         elif self.raise_upon_error and self.excep:
             raise self.excep
 
-    def event_contents(self, docs, full_event=False):
+    def doc_contents(self, docs, full_event=False):
```

Comment:
Internal API change (since we can pull from any document, it would be nice to rename this to something more general)